doc/source/Platforms.rst

```diff
@@ -661,28 +661,35 @@ Remember to activate the ``lua`` module environment and have MacTeX in your sear
    spack config add "packages:all:providers:mpi:[openmpi@4.1.4]"
    spack config add "packages:all:compiler:[apple-clang@13.1.6]"
 
-7. Optionally, edit site config files and common config files, for example to remove duplicate versions of external packages that are unwanted, add specs in ``envs/jedi-ufs.mymacos/spack.yaml``, etc.
+7. Edit the main config file for the environment and adjust the compiler matrix to match the compilers for macOS, as above:
```

Comment:
Note to self. Add "if applicable" or something similar, since not all environments have this compiler matrix function (in fact, only `unified-dev` has).